### PR TITLE
Kwil-admin setup testnet command fix

### PR DESCRIPTION
**Pull request by charithabandi**

### CODE REVIEW

File: cmd/kwil-admin/nodecfg/toml_test.go [hunk 1]
```diff
@@ -74,34 +75,62 @@ func Test_IncrementingPorts(t *testing.T) {
...
 		want   string
+		fail   bool
 	}
 
 	testcases := []testcase{
 		{
 			input:  "localhost:26656",
 			amount: 1,
-			want:   "localhost:26657",
+			want:   "tcp://localhost:26657",
```
Comment (jchappelow) [inline cmd/kwil-admin/nodecfg/toml_test.go:85]: Hmmm this seems undesirable for our service settings, right?  I'm surprised it's works for the http setting for example. 

"tcp://" is not a real scheme, just a strange cometbft thing. Seems unfortunate to adopt that for our http rpc service. 

Seem weird or ok?

Comment (charithabandi) [reply cmd/kwil-admin/nodecfg/toml_test.go:85]: Yeah, agreed. Let me fix that

Comment (charithabandi) [reply cmd/kwil-admin/nodecfg/toml_test.go:85]: Yeah, agreed. Let me fix that

File: cmd/kwil-admin/nodecfg/generate.go [hunk 2]
```diff
@@ -546,7 +556,7 @@ func incrementPort(incoming string, amt int) (string, error) {
 	// Reconstruct and return the new URL
 	newUrl := net.JoinHostPort(host, strconv.Itoa(port))
 
-	if res.Scheme != "localhost" {
+	if schemaExists {
```
Comment (jchappelow) [inline cmd/kwil-admin/nodecfg/generate.go:559]: Nice.  I like this solution way better.  The scope of the changes logically should be limited to nodecfg (other than exporting `HasScheme`).
Will test in the a.m., but LGTM.

**Review by jchappelow — APPROVED** [review]

**Pull request merged**